crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -79,7 +287,7 @@ mod tests {
     #[test]
     fn test_parse_property_no_type() {
         let mut input: ItemStruct = parse_quote! {
-            #[qproperty(T)]
+            #[qproperty(num)]
```

Comment:
any reason for this change? it doesn't seem to be relevant?

Reply:
or should the name of the test be `_no_name` rather than `_no_type` :-)

Reply:
because the test is for testing no type, not no identifier, it was a duplicate of the test above it and existed already

Reply:
Maybe the test should be removed as i don't think it can tell the difference between a name and a type, what we are really testing here is if there is 1 argument does it fail as we require 2.

Maybe instead this test could check that 0 arguments fails instead? So we have the test above for checking 1 argument fails and change this test for 0 arguments.

Reply:
done